.github/workflows/rust.yml

```diff
@@ -1,3 +1,9 @@
+# Copyright (see LICENSE.md) - 2022
```

Comment:
Copyright should be your name (c) 2022. You wrote it.

Reply:
thanks :) done too